airflow/www_rbac/views.py

```diff
@@ -2345,29 +2346,18 @@ class VersionView(AirflowBaseView):
 
     @expose('/version')
     @has_access
+    
     def version(self):
-        try:
-            airflow_version = airflow.__version__
-        except Exception as e:
-            airflow_version = None
-            logging.error(e)
 
-        # Get the Git repo and git hash
-        git_version = None
-        try:
-            with open(os.path.join(*[settings.AIRFLOW_HOME,
-                                   'airflow', 'git_version'])) as f:
-                git_version = f.readline()
-        except Exception as e:
-            logging.error(e)
-
-        # Render information
-        title = "Version Info"
+        data = None
+        filepath = os.path.join((os.path.abspath(os.path.dirname(__file__))), '../changelog.yaml')
```

Comment:
Keep this path as an constant in `airflow/settings.py`.

Reply:
Where in settings.py do I have to declare the path variable and how to include it?

Reply:
See how these constants are initialised: https://github.com/coutureai/RaWorkflowOrchestrator/blob/cab9d84b85d76596649e4fc45b01c77f1bdf5837/airflow/settings.py#L151